README.md

```diff
@@ -11,7 +11,7 @@ plugins {
 
 jxbrowser {
     // The JxBrowser version. A mandatory field.
-    version = "7.35.2"
+    version = "7.36"
```

Comment:
The problem with the fixed version of JxBrowser in README is that it would require updating it every time we release a new version of JxBrowser. It's a small and easy task, but it is likely that it would be forgotten or skipped even if we add it in a release checklist.

Even if it won't be skipped, we still can help people to find out the latest version of JxBrowser.

How about adding adding the following comment line?

```java
// Obtain the latest release version number at https://teamdev.com/jxbrowser/.
```